osu.Game/Screens/Menu/Disclaimer.cs

```diff
@@ -102,6 +108,8 @@ protected override void LoadComplete()
         {
             base.LoadComplete();
             LoadComponentAsync(intro = new Intro());
+
+            api.LocalUser.BindValueChanged(userChanged, true);
```

Comment:
Is there a reason why you're not creating a local bindable? This sort of usage of bindables (binding to the outside's valuechanged) is generally seen as unsafe.